src/client/chrome/events.js

```diff
@@ -74,8 +76,7 @@ async function paused({
   if (clientType == "chrome") {
     pageAgent.configureOverlay({ message: "Paused in debugger.html" });
   }
-
-  await actions.paused({ frame, why, frames, loadedObjects });
+  await actions.paused({ thread: "root", frame, why, frames, loadedObjects });
```

Comment:
@jasonLaster not sure if this the best thread value, couldn't find something better, your thoughts?

Reply:
haha - it's fine...